### ISSUE TITLE
[Documentation] Improve/Clarify on https://www.graalvm.org/reference-manual/native-image/#build-a-shared-library

### ISSUE THREAD

**Issue by rubyFeedback** [oracle/graal#4166]
Hey there GraalVM seekers,

I am reading up on some more GraalVM stuff, as I was the last ~days - lots of
 information available out there which is great. Hope that 2022 will be an 
even better year for GraalVM. Hopefully many more blog articles with useful
knowledge and snippets to learn from!

In regards to the issue here, I am reading up on that URL:

https://www.graalvm.org/reference-manual/native-image/#build-a-shared-library

However had, halfway in the page I got lost a bit. Not sure whether
it is because I am tired (it's quite late here), or whether I was
not concentrated or perhaps the quality of the article not being
meant for my (feeble) brain level.

For instance it shows the following as of right now:

    @CEntryPoint static int add(IsolateThread thread, int a, int b) {
      return a + b;
    }

But I am not entirely certain how to use that specific example.

Would it be possible if a small example file could be shown
that is complete? Either a full download, or a full example
with the code (I can copy/paste just fine, that is no problem).

I'd love to try the example, and see how that can work with a
shared example. (I am still somewhat a java noob; while I can
get things done, at this point I simply don't know a lot, so
shared libraries in regards to Java are a bit over my head, 
which is why I think it would be nice to improve that
particular part of the documentation.)

So, TL;DR suggestion is:

Please extend the "native-image class [libraryname] --shared"
example with:

a) a specific standalone code example, not just
"@CEntryPoint static int add(IsolateThread thread, int a, int b) {"

and

b) an example how to get this to work/compile.

Could be just a simple hello world example again, just with
the shared library part or something (e. g. the function
that is mentioned add(), but I don't know how to get this
to work right now).

Thanks for reading and potentially considering!

**Comment by rodrigar-mx** [comment]
Hi @rubyFeedback. Thanks for using graalvm. Regarding shared libraries for native methods you can find all the steps here https://www.graalvm.org/reference-manual/native-image/ImplementingNativeMethodsInJavaWithSVM/. Please take a look into it and let me know if you have any questions.

**Comment by rodrigar-mx** [comment]
Closing this issue because there has been no activity during last months.
